e2e/main.e2e.spec.ts

```diff
@@ -40,69 +43,54 @@ test.describe.serial('Main App Test', () => {
   });
 
   test('should land on General page', async() => {
-    mainTitle = page.locator(mainTitleSelector);
+    const navPage = new NavPage(page);
 
-    await expect(mainTitle).toHaveText('Welcome to Rancher Desktop');
+    await navPage.getPageTitle('Welcome to Rancher Desktop');
```

Comment:
```suggestion
    await expect(navPage.pageTitle).toHaveText('Welcome to Rancher Desktop');
```

Looking at [docs](https://playwright.dev/docs/test-pom), it doesn't look like page objects are expected to embed assertions, other than as timing points…

Reply:
That interesting. It actually works, embedding asserting straight in the page model... but after you comments I realize it seems quite weird though, too much unnecessary abstraction.
thanks for the heads up, I'll change that.